src/module_3/module3_statistical_learning.md

```diff
@@ -0,0 +1,1368 @@
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+      <td>9.0</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2808027644036</td>
+      <td>3466586718340</td>
+      <td>2020-10-05 17:59:51</td>
+      <td>2020-10-05</td>
+      <td>2</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.081052</td>
+      <td>0.053512</td>
+      <td>clearspring</td>
+      <td>0.000000</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+      <td>6.0</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2808099078276</td>
+      <td>3481384026244</td>
+      <td>2020-10-05 20:08:53</td>
+      <td>2020-10-05</td>
+      <td>4</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.081052</td>
+      <td>0.053512</td>
+      <td>clearspring</td>
+      <td>0.000000</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+      <td>9.0</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2808393957508</td>
+      <td>3291363377284</td>
+      <td>2020-10-06 08:57:59</td>
+      <td>2020-10-06</td>
+      <td>2</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.081052</td>
+      <td>0.053512</td>
+      <td>clearspring</td>
+      <td>0.038462</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+      <td>13.0</td>
+    </tr>
+    <tr>
+      <th>5</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2808434524292</td>
+      <td>3479090790532</td>
+      <td>2020-10-06 10:50:23</td>
+      <td>2020-10-06</td>
+      <td>3</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.081052</td>
+      <td>0.053512</td>
+      <td>clearspring</td>
+      <td>0.038462</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+      <td>7.0</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+
+```python
+dfSummary(df_orders)
```

Comment:
the output of this function is very confusing, there are much better things to do summary of data frames, you can start by a simple `describe` or can go to more sophisticated things like `from pandas_profiling import ProfileReport`

Reply:
will try out `from pandas_profiling import ProfileReport`, first time I've heard of it